libraries/ui/src/Nav.test.tsx

```diff
@@ -23,9 +23,36 @@ describe('Nav', () => {
           { title: 'Course 1', href: '/course1' },
         ]}
       >
-        <p>This is the yield</p>
+        <p id="yield-element">This is the yield</p>
       </Nav>,
     );
+    const yieldElement = container.querySelector('#yield-element');
+    expect(yieldElement).not.toBeNull();
+
     expect(container).toMatchSnapshot();
   });
+
+  test('clicking the hamburger menu toggles nav__drawer max height above 1', async () => {
```

Comment:
can we rephrase this? Something like
```suggestion
  `test('that when the menu collapses it is still visible on all devices', async () => {
```

Reply:
Hm, that specific wording isn't exactly what this is testing, but if you're thinking about making it more semantically describe what is supposed to happen for the user then how about `'clicking the hamburger button expands the mobile nav drawer'`

(^I'll go with this wording by default)